service_crategen/src/commands/generate/mod.rs

```diff
@@ -230,9 +237,13 @@ pub use custom::*;
                     .open(&custom_mod_file_path)
                     .expect("Unable to write mod.rs");
             }
+            if log_enabled!(Debug) {
```

Comment:
Why do we branch here?

`debug!()` won't actually log to STDOUT if we don't specify that level (and default is info). Curious what this branch actually gets us.

Reply:
Good point.  I copied it from our use of the logger in the request dispatcher, where it's expensive to do the work inside the `if` block.  Here it really doesn't matter and could go away.